packages/SwingSet/package.json

```diff
@@ -58,10 +60,6 @@
     "semver": "^6.3.0",
     "tmp": "^0.2.1"
   },
-  "peerDependencies": {
```

Comment:
we move these out of `peerDependencies`? here's why they were put there: https://github.com/Agoric/agoric-sdk/pull/5232

Reply:
After having read that issue, I'm willing to leave `ava` in `peerDependencies`.  However, IMO `@endo/ses-ava` should be a normal dependency since it is an implementation detail of swingset-vat/tools.  Otherwise, SwingSet initialisation fails during Endo bundling when running from the `@agoric/swingset-vat` NPM package (as verified by the integration test).

Reply:
> @endo/ses-ava should be a normal dependency since it is an implementation detail of swingset-vat/tools.

Ah yes, agreed.